assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -95,7 +95,7 @@ function MyStories() {
   const resultsLabel = useDashboardResultsLabel({
     isActiveSearch: Boolean(search.keyword),
     currentFilter: filter.value,
-    totalResults: totalStories,
+    totalResults: totalStoriesByStatus?.all,
```

Comment:
Here is where I think we'd be briefly flashing `undefined`

Reply:
I think it's actually because while reviewing my own code I updated the `HeaderToggleButtons` to check for keys on totalStoriesByStatus and forgot to add `.length` (so i fixed that). _However_ you bring up a great point that we would do well to not render that 'Viewing all ____' until we have a current status to show so I added a check in the `useDashboardResultsLabel` for the `currentFilter` to exist otherwise default to an empty string. The `totalResults` there isn't used at all until `isActiveSearch` is true. 

Regardless, sorry about this - my changes must have not taken effect when I tested locally before submitting for review and I thought it was fine.